test/it/uk/gov/hmrc/exports/repositories/UpdateSubmissionsTransactionalOpsISpec.scala

```diff
@@ -337,14 +339,14 @@ class UpdateSubmissionsTransactionalOpsISpec extends IntegrationTestSpec {
               .futureValue
               .isRight mustBe true
 
-            val result = intercept[InternalServerException] {
-              await {
-                transactionalOps
-                  .updateSubmissionAndNotifications(amendmentAction.id, List(notificationForAmendment), testSubmission)
+            val resultOfOp =
+              transactionalOps.updateSubmissionAndNotifications(amendmentAction.id, List(notificationForAmendment), testSubmission).transformWith {
+                case Success(_)                            => Future(s"The operation was successful on SubmissionStatus($status)")
+                case Failure(ise: InternalServerException) => Future(ise.message)
+                case Failure(exc) => Future(s"A ${exc.getClass.getName} is not the test-expected exception. Error was: ${exc.getMessage}")
```

Comment:
Due to random failures of this test, I changed hoping to get more information in case it fails again.